extensions/git/src/git.ts

```diff
@@ -661,6 +670,7 @@ export enum ForcePushMode {
 }
 
 export class Repository {
+	private filters = new Set<string>();
```

Comment:
I personally prefer this initialization done in the `constructor`, but :man_shrugging: I believe this would also work sufficiently in TypeScript

Reply:
I don't have any preferences on this, happy to use whatever standard is in VS code base (which to be honest I haven't checked).